Azure/sonic-utilities
# [sonic-installer] migrate SONiC packages

**- What I did**

Implement package migration logic.

**- How I did it**

According to App Ext. design document.

**- How to verify it**

Install some extension, performs S2S upgrade and verify that extension is installed in newer image.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**
